api/controllers/restaurantController.js

```diff
@@ -57,13 +58,27 @@ const getRestaurants = async (req, res) => {
 };
 
 const getRestaurant = async (req, res) => {
-  const { restaurantId } = req.params;
-  const restaurant = await Restaurant.findByPk(restaurantId, {include: ['cuisines', 'reviews']});
-  if (restaurant) {
-    res.send(restaurant);
-  } else {
-    res.send(mockRestaurants[0]);
-  }
+  const { APIID } = req.params;
```

Comment:
Can't change params to APIID because that is being used to handle restaurants updates and deletes. If you want to pass yelp id in the request, then use a query parameter.

``` suggestion
  const { restaurantId } = req.params;
  const { yelpId } = req.query;
```
The restaurant column has a yelpId column now in PR #22. Instead, add these axios requests in addition to what the restaurant table returns.